### PR TITLE
Remove pabot tests

### PR DESCRIPTION
**Pull request by blag**
As mentioned on Slack, remove the RobotFM tests that are inconsistently failing.

### CODE REVIEW

File: actions/workflows/st2_e2e_tests.yaml [hunk 1]
```diff
@@ -95,42 +95,42 @@ workflows:
...
+            # pabot_chatops_tests:
+            #     action: core.remote
+            #     input:
+            #         hosts: <% $.host_ip %>
+            #         env: <% $.st2_cli_env %>
+            #         cmd: "cd /tmp/st2tests && . ./venv/bin/activate && bash /tmp/st2tests/robot_parallel_tests.sh '--processes 1 robotfm_tests/chatopsCI/' 1"
+            #         timeout: 180
+            #     retry:
+            #         count: 2
+            #         delay: 10
```
Comment (m4dcoder) [inline actions/workflows/st2_e2e_tests.yaml]: I would remove them altogether.  If we want it back, we can always see earlier commits.

**Review by m4dcoder — CHANGES_REQUESTED** [review]
Minor change request but otherwise LGTM.

File: actions/workflows/st2_e2e_tests.yaml (hunk 1 above)
Comment (m4dcoder) [reply actions/workflows/st2_e2e_tests.yaml]: I don't like commenting out codes. This leaves for other people to clean up.

**Review by m4dcoder — APPROVED** [review]

File: actions/workflows/st2_e2e_tests.yaml (hunk 1 above)
Comment (blag) [reply actions/workflows/st2_e2e_tests.yaml]: Yeah, I wouldn't have left it in if I wasn't planning on ripping them out soonishly.

But either way, fixed.

**Pull request merged**